### DISCUSSION TITLE
ERROR: Could not install packages due to an OSError

### DISCUSSION THREAD

**Discussion by Alzwei** [carson-katri/dream-textures#482] [General]
Try to install the dependency, but Windows Defender deletes the file "huggingface_hub-0.11.1-py3-none-any.whl".
Windows Defender report: 
Detected: Trojan:Script/Sabsik.TE.A!ml

Affected items:  file: C:\Users\alex_\AppData\Local\Temp\pip-unpack-fpqllguv\huggingface_hub-0.11.1-py3-none-any.whl

Console:

Collecting huggingface_hub
  Downloading huggingface_hub-0.11.1-py3-none-any.whl (182 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 182.4/182.4 kB 1.2 MB/s eta 0:00:00
ERROR: Could not install packages due to an OSError: [Errno 22] Invalid argument: 'C:\\Users\\alex_\\AppData\\Local\\Temp\\pip-unpack-fpqllguv\\huggingface_hub-0.11.1-py3-none-any.whl'

**Comment by carson-katri** [comment]
You don't need to install dependencies, just download a prebuilt release from the Releases section.
